curvefs/conf/client.conf

```diff
@@ -23,7 +23,7 @@ excutorOpt.minRetryTimesForceTimeoutBackoff=5
 excutorOpt.maxRetryTimesBeforeConsiderSuspend=20
 
 #spaceserver
-spaceserver.spaceaddr=127.0.0.1:19999
+spaceserver.spaceaddr=127.0.0.1:19999  # __TMPL__ {{ groups.space | join_peer(hostvars, "space_listen_port") }} __TMPL__
```

Comment:
can `__TMPL__` be replaced by a more meaningful word? currently, it's confusing.

Reply:
> can `__TMPL__` be replaced by a more meaningful word? currently, it's confusing.

The `__TMPL__` is the abbreviation of `template`, maybe we can use `__TEMPLATE__` for understandable.